app/templates/views/service-settings/email-reply-to/_verify-updates.html

```diff
@@ -0,0 +1,64 @@
+{% from "components/banner.html" import banner, banner_wrapper %}
+{% from "components/textbox.html" import textbox %}
+{% from "components/checkbox.html" import checkbox %}
+{% from "components/page-header.html" import page_header %}
+{% from "components/page-footer.html" import page_footer %}
+{% from "components/form.html" import form_wrapper %}
+
+<div class="ajax-block-container">
+  {% if verification_status == "pending" %}
+    <p>
+      We’re checking that ‘{{ reply_to_email_address }}’ is a real email address.
+    </p>
+    <p>
+  <span class='loading-indicator'>This can take a minute </span>
+    </p>
+
+    <p>
+      <a href="{{ url_for('main.service_verify_reply_to_address', service_id=service_id, notification_id=notification_id) + "?is_default={}".format(is_default) }}"">Refresh</a>
```

Comment:
`url_for` will generate the query string for you:
```suggestion
      <a href="{{ url_for('main.service_verify_reply_to_address', service_id=service_id, notification_id=notification_id, is_default=is_default) }}">Refresh</a>
```